app/views/users/_user_groups.erb

```diff
@@ -1,8 +1,11 @@
 <div class='groupContainer'>
-		<header><%= group.name %></br><small><strong>EVENTS</strong></small></header>
```

Comment:
header tag is indicated but with no use...... use of <br> tags should not be used ... elements should be defined as block, inline-block or inline. <strong>, <small> tags are supposed to be used for emphasis not styling. I have replaced these with the proper css styles.